deployment/docker/build.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
-cd ../../
-docker build -t kartoza/projecta-uwsgi -f deployment/docker/Dockerfile .
+pushd ../../
+docker build -t kartoza/projecta-uwsgi -f deployment/docker/Dockerfile --target prod .
+popd
```

Comment:
Now we specify the target `prod`

This script is only used if you want to build the image manually (legacy reason).

A better way is to just use `docker-compose build`.